.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Report a bug encountered while operating Kubernetes
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
+
+/kind bug
```

Comment:
For all the commands, like `/kind bug` can we do something similar to [this](https://github.com/kubernetes/org/blame/master/.github/ISSUE_TEMPLATE/membership.md#L30-L31) where we put them at the bottom with `<!-- DO NOT EDIT BELOW THIS LINE -->` proceeding them?

Reply:
Good idea!